packages/angular/package.json

```diff
@@ -9,7 +9,8 @@
     "build": "ng build --configuration production",
     "test": "jest",
     "test:watch": "yarn test --watch",
-    "lint": "ng lint"
+    "lint": "yarn typecheck && eslint . --ext .js,.jsx,.ts,.tsx",
```

Comment:
Good with adding `typecheck` here but let's continue using the built in `ng lint` script. Angular has a lot going on under the hood ("magic") that we don't have awareness in to